stylecomponents/card.css

```diff
@@ -0,0 +1,32 @@
+.card {
+  max-width: 800px;
+  display: flex;
+  flex-flow: column wrap;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  border: 1px solid var(--primarycolor);
+  margin: 1rem;
+  padding: 1.2rem;
+}
+
+.card__bookmark {
+  position: absolute;
+  top: -15px;
+  right: 20px;
+  height: 2rem;
+  padding: 0px;
+}
+
+.card__bookmark__filled {
```

Comment:
You updated the name in your HTML files correctly. No you also need to change it here too. :)